### PR TITLE
Added homework 'Company'

### PR DESCRIPTION
**Pull request by Ivanko-Mosiychuk**

### CODE REVIEW

File: src/main/java/com/justcode/Main.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+public class Main {
+    static private CompanyAccount account = new CompanyAccount(new Company("JustCode"));
+
+    public static void main(String[] args) {
+        addTestEmployees();
+        account.printEmployeesName();
+        account.printEmployeesData(account.getSortedEmployeesBySalary(1500, 3500));
+        try {
+            account.printEmployeesData(account.getAllPermanentEmployee());
+        } catch (NotFoundAnyEmployeeException e) {
```
Comment (nmelnyk-corevalue) [inline src/main/java/com/justcode/Main.java:17]: NotFoundAnyEmployeeException can we put try-catch inside the method getAllPermanentEmployee

File: src/main/java/com/justcode/company/Company.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    public Company(String name) {
+        this.name = name;
+        this.employees = new ArrayList<>();
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public List<Employee> getEmployees() throws NotFoundAnyEmployeeException {
```
Comment (nmelnyk-corevalue) [inline src/main/java/com/justcode/company/Company.java:23]: throws NotFoundAnyEmployeeException is redundant here

File: src/main/java/com/justcode/company/CompanyAccount.java [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+    //Повертаж список всіх постійних працівників
+    public List<Employee> getAllPermanentEmployee() throws NotFoundAnyEmployeeException {
+        sortedEmployees.clear();
+        try {
+            for (Employee employee : company.getEmployees()) {
+                if (!employee.isHired()) {
+                    sortedEmployees.add(employee);
+                }
+            }
+        } catch (NotFoundAnyEmployeeException e) {
```
Comment (nmelnyk-corevalue) [inline src/main/java/com/justcode/company/CompanyAccount.java:53]: maybe this is redundant here

File: src/main/java/com/justcode/company/CompanyAccount.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            e.printStackTrace();
+        }
+
+        if (sortedEmployees.isEmpty()) {
```
Comment (nmelnyk-corevalue) [inline src/main/java/com/justcode/company/CompanyAccount.java:57]: great!

**Review by nmelnyk-corevalue — APPROVED** [review]

File: src/main/java/com/justcode/company/Company.java (hunk 2 above)
Comment (Ivanko-Mosiychuk) [reply src/main/java/com/justcode/company/Company.java:23]: Do you want I use way like this:
  try{
            throw new MyException(); 
        } 
        catch (MyException ex) 
        { 
            System.out.println("Caught"); 
            System.out.println(ex.getMessage()); 
        }

**Pull request closed without merging**